pages/vi/profiles/abdusamaraie.md

```diff
@@ -0,0 +1,48 @@
+# Welcome to my Profile
+___________
+
+**location:** US 
+
+**Time Zone:** EST 
+
+**OS:** Ubuntu 18.04
+
+
+![alt MY Country <](https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/%D8%B3%D8%A7%D8%AD%D8%A9-%D8%A7%D9%84%D8%AA%D8%AC%D8%B1%D9%8A%D8%B1.png/400px-%D8%B3%D8%A7%D8%AD%D8%A9-%D8%A7%D9%84%D8%AA%D8%AC%D8%B1%D9%8A%D8%B1.png "Photo of my home town")
+
+## About Me:
+
+My name is Abdu Samaraie and I am a Computer Science student. I am very passionate programmer, and I like learning new technologies that will help me excel in the field of Software Engineering. In my freetime, I am constantly learning by watching toturials on Youtube.
+
+I graduated for the University at Buffalo This year (2019)
+
+
+![Image logo](https://www.suny.edu/media/suny/content-assets/images/campus-profiles/logos/buffalo.jpg)
+
+Add me on [LinkedIn](https://www.linkedin.com/in/abdu-samaraie/)
+
+
+#### My Favorite Programming Languages:
+* Python
+* JavaScript
+* SQL
+
+
+`Best quote`
+>  The greatest of richness is the richness of the soul.
+>                                                      _Prophet Muhammad (peace be upon him)
```

Comment:
is this `_` intended to be there or was added for some makdown?

Reply:
was intended, for citation